.travis.yml

```diff
@@ -1,22 +1,28 @@
+sudo: false
 language: android
 android:
   components:
-    # Uncomment the lines below if you want to
-    # use the latest revision of Android SDK Tools
     - tools
     - platform-tools
-    - tools
-
-    # The BuildTools version used by your project
     - build-tools-27.0.2
-
-    # The SDK version used to compile your project
     - android-27
+    - android-22
```

Comment:
We should remember to inform everyone of this change since most people are still building with sdk 27 @jusleg

Reply:
what do you mean? The project is building for 27 but the emulator is running with 22. There shouldn't be any problem as the app is backward compatible to api 16